plugin/src/main/java/io/jenkins/plugins/remotingkafka/GlobalKafkaConfiguration.java

```diff
@@ -46,37 +46,65 @@ public GlobalKafkaConfiguration() {
     }
 
     public static GlobalKafkaConfiguration get() {
```

Comment:
Add `@Symbol` to this class so that you get better name than the default `globalkafkaconfiguration`